protocol-units/execution/maptos/util/src/config/common.rs

```diff
@@ -71,6 +72,36 @@ env_default!(
 	"0.0.0.0".to_string()
 );
 
+// The default Da Sequencer connection url
+env_default!(
+	default_da_sequencer_connection_url,
+	"MAPTOS_FIN_VIEW_DA_SEQUENCER_CONNECTION_URL",
```

Comment:
I still don't know what fin view means. If it's no longer important to our system, can we change it and/or remove it?

Reply:
`Fin` is for finality (block finalization). The fin view is the view of the ledger db after finality. Currently, we're only doing execution and no settlement. So there's no finality. If we activate the settlement the finality will be activated, and you can get the account db view when block are finalized. @mzabaluev work on it this summer and mostly on the block reversion. If the settlement is rejected, the executed block must be reverted.
I don't know what  `MAPTOS_FIN_VIEW_DA_SEQUENCER_CONNECTION_URL` is use for because the finality is after execution, so I don't see why it needs a Da connection.
It is used nowhere in the code, so it can be removed.
I think it will be part of the cleaning at the end of the Da Sequencer dev.